### PR TITLE
Add `destinationPath` option

### PR DESCRIPTION
**Pull request by drag0s**
If this option is specified, it would allow a user to choose video's destination path. Else, destination path would still be the path generated by [tempy](https://github.com/sindresorhus/tempy).

### CODE REVIEW

**Comment by sindresorhus** [comment]
What's your use-case for this?

File: readme.md [hunk 1]
```diff
@@ -150,6 +150,13 @@ The `hevc` codec requires macOS 10.13 or newer. A computer with Intel 6th genera
 
 The [`proRes422` and `proRes4444`](https://documentation.apple.com/en/finalcutpro/professionalformatsandworkflows/index.html#chapter=10%26section=2%26tasks=true) codecs are uncompressed data. They will create huge files.
 
+#### destinationPath
+
+Type: `string`<br>
+Default: A `.mp4` file in a temporary directory
+
+Destination file path for the video.
+
```
Comment (sindresorhus) [inline readme.md]: It should be noted that it's better to not set this and rather move the file when the export is done, so users don't see a incomplete file during recording.

**Comment by timwedde** [comment]
I know this PR is already a bit older however I, too, would be interested in this addition.
I can't speak for anyone else of course but my personal use case for this is relatively simple: My main drive is quite small and already occupied by other vital stuff. Not being able to change where the original recording saves to is a major hindrance when you only have very little space left.
Meanwhile I have an entirely unused 1TB SSD mounted anyway, which I could record to without a problem if this were implemented.

File: readme.md (hunk 1 above)
Comment (sindresorhus) [reply readme.md]: Or set this to a temp directory on a different disk if it's set for space reasons.

**Comment by sindresorhus** [comment]
Closing for lack of response. Someone else can continue it if they need it: https://github.com/wulkano/aperture-node/issues/9

**Comment by alectrocute** [comment]
Continued in #11. I argue the use case of moving a file once it's done may be trouble – like @timwedde mentioned, if the temporary folder isn't an option for whatever reason, drive space etc., then it may be best to allow the user to explicitly set the processing/final location.

**Comment by alectrocute** [comment]
For my specific use case, I'm saving the file externally, so this makes sense. @sindresorhus, thoughts?

**Comment by sindresorhus** [comment]
I'm fine with adding this, but the docs should mention the different use-cases for the option.

**Comment by alectrocute** [comment]
> I'm fine with adding this, but the docs should mention the different use-cases for the option.

Great! I'll append, squash and prepare this.

**Pull request closed without merging**